real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/RBSMNode.java

```diff
@@ -56,43 +67,87 @@
 	
 	Publisher statePub_enc;
 	Publisher statePub_pot;
+	
+	Subscriber messageSub_angle;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1384444228/issues/source?bid=2938665&fileBranchId=2938741#l71)